pycodestyle_run.py

```diff
@@ -18,7 +18,7 @@ class TestCodeFormat(unittest.TestCase):
     def test_conformance(self):
         """Test that we conform to PEP-8."""
         style = pycodestyle.StyleGuide(config_file='pycodestyle.cfg')
-        result = style.check_files(['./src', './tests'])
+        result = style.check_files(['./src/evagram/database', './tests'])
```

Comment:
Why hardcode "database"? Is there a reason you'd want pycodestyle_run to skip over sra/evagram/website?

Reply:
Thanks for catching this, I originally skipped over the website directory because the React application had a `node_modules` folder that contained some Python scripts not written by me that would fail the pycodestyle. The `node_modules` folder is in gitignore, so it should not affect the Python Coding Norms action. However, I now realize that the action is also not checking files under the Django application, so I will make that change in the next commit.